QSB/Events/EventHandler.cs

```diff
@@ -44,19 +44,15 @@ private void OnServerReceiveMessage(EventMessage message)
 
         private void OnClientReceiveMessage(EventMessage message)
         {
-            if (message.SenderId == PlayerRegistry.LocalPlayer.NetId)
-            {
-                return;
-            }
```

Comment:
should still keep this?

Reply:
I did, it just moved to below the first switch